Lib/test/test_os.py

```diff
@@ -2200,16 +2199,12 @@ def test_remove_directory_link_to_missing_target(self):
         #  was created with target_is_dir==True.
         os.remove(self.missing_link)
 
-    @unittest.skip("currently fails; consider for improvement")
```

Comment:
Why is this no longer skipped: has this behavior been implemented?

Likewise for the next test below.